### ISSUE TITLE
Writer a separate operator to install KubeAGI with a CRD

### ISSUE THREAD

**Issue by nkwangleiGIT** [kubeagi/arcadia#949]
we should define a CRD to make KubeAGI installation quite easy, so we can install u4a/cluster/kubeagi-system together without a deep understanding how they work.

**Comment by bjwswang** [comment]
@nkwangleiGIT should we also install cluster-component,u4a-component,kubebb-component in this operator?

**Comment by nkwangleiGIT** [comment]
yes, one command install using the CRD + controller, cover all components required.
